### PR TITLE
CPR-312-appinsight-message-received

### PR DESCRIPTION
**Pull request by JunaedAhmedMOJ**
Removed court specific message and added them into general message received

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/court/CourtEventProcessor.kt [hunk 1]
```diff
@@ -59,7 +59,7 @@ class CourtEventProcessor(
     uniqueDefendants.forEach { defendant ->
       val person = Person.from(defendant)
       telemetryService.trackEvent(
-        COURT_MESSAGE_RECEIVED,
+        TelemetryEventType.MESSAGE_RECEIVED,
```
Comment (H-Iwanejko) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/court/CourtEventProcessor.kt:62]: Better if we just leave it as `COURT_MESSAGE_RECEIVED `

File: src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/court/CourtEventProcessor.kt [hunk 2]
```diff
@@ -81,7 +81,7 @@ class CourtEventProcessor(
     val person = Person.from(libraHearingEvent)
 
     telemetryService.trackEvent(
-      COURT_MESSAGE_RECEIVED,
```
Comment (H-Iwanejko) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/court/CourtEventProcessor.kt:84]: Same here

File: src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt [hunk 3]
```diff
@@ -112,7 +112,7 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
     } matches { it == 0 }
 
     checkTelemetry(
-      CPR_RECORD_CREATED,
+      TelemetryEventType.CPR_RECORD_CREATED,
```
Comment (H-Iwanejko) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt:115]: Same for these

**Review by H-Iwanejko — COMMENTED** [review]
Just a couple tidy ups but other than that 👍

**Review by markreesmoj — APPROVED** [review]
Requested changes made 👍

**Pull request merged**